src/Provider/Vkontakte.php

```diff
@@ -21,53 +21,51 @@
  *       'callback'  => Hybridauth\HttpClient\Util::getCurrentUrl(),
  *       'keys'      => ['id' => '', 'secret' => ''],
  *   ];
- *
```

Comment:
I don't think we need to change something here in this description.

Reply:
@ApacheEx why?